easybuild/easyblocks/i/impi.py

```diff
@@ -186,18 +196,22 @@ def make_module_req_guess(self):
                 'LIBRARY_PATH': lib_dirs,
                 'MANPATH': ['man'],
                 'CPATH': include_dirs,
-                'MIC_LD_LIBRARY_PATH' : ['mic/lib'],
+                'MIC_LD_LIBRARY_PATH': ['mic/lib'],
             }
         else:
-            lib_dirs = ['lib/em64t', 'lib64']
-            include_dirs = ['include64']
+            # Keep release_mt and release in front, to give priority to the possible symlinks in intel64/lib.
+            # IntelMPI 2019 changed the default library to be the non-mt version.
+            lib_dirs = ['intel64/%s' % x for x in ['lib/release_mt', 'lib/release', 'lib', 'libfabric/lib']] +
+                    ['lib/em64t', 'lib64']
```

Comment:
@damianam As discussed, please clearly discriminate between the paths that should be consider for `impi` 2019.x and the older versions.